pyface/ui/qt4/util/modal_dialog_tester.py

```diff
@@ -247,6 +247,22 @@ def click_widget(self, text, type_=QtGui.QPushButton):
         )
         widget.click()
 
+    def click_ok(self):
```

Comment:
How about using a dictionary 
```python
BUTTONS = {'ok': 'OK', 'OK', 'yes': '&Yes'}
```
it would be easier to extend later to container buttons like 'Next', 'Continue', 'Stop'

It would be easier to extent later